docs/howto_guides/api_gui.md

```diff
@@ -0,0 +1,165 @@
+# Using the NMDC API Graphical User Interface (GUI)
+
+## Retrieving and Submitting Metadata using the ___Find___ and ___Metadata___ API Endpoints
+
+Metadata describing NMDC data (e.g. studies, biosamples, data objects, etc.) may be retrieved or submittied with GET and POST requests, respectively, using the **[NMDC API Graphical User Interface (GUI)](https://api.microbiomedata.org/docs#/)** The API GUI provides an user interface for programmatic access to the NMDC data portal without needing to use the Command Line.
```

Comment:
> user interface for programmatic access

"programmatic access" is generally non-graphical and non-interactive, i.e. via python scripting, shell scripting with `curl`, etc. I'd emphasize that what this API GUI provides is a guided interface for (1) performing highly granular and targeted queries directly -- ones that may not be directly supported by the Data Portal UI (yet), and (2) exploring query capabilities interactively and in the process providing code snippets to adapt a data-consumer workflow to using programmatic access, i.e. a user can copy the formed URLs, `curl` commands, etc. to help them construct scripts.

Reply:
Thank you both. I included your feedback in [7372ee3f3953633f8540a8860487803c316e9a99](https://github.com/microbiomedata/NMDC_documentation/pull/27/commits/7372ee3f3953633f8540a8860487803c316e9a99)